app/schemas/com.uxstate.launchpad.data.local.LaunchDatabase/1.json

```diff
@@ -0,0 +1,162 @@
+{
```

Comment:
I am interested in this. Do you write this manually or is there a tool to write this?